coffee-rest/src/main/java/hu/icellmobilsoft/coffee/rest/log/RequestResponseLogger.java

```diff
@@ -406,8 +406,9 @@ public String printEntity(Object entity, Integer maxLogSize, String prefix, bool
         } else {
             entityText = entity.toString();
         }
-        if (maxLogSize != null && entityText != null && entityText.length() > maxLogSize.intValue()) {
-            entityText = StringUtils.substring(entityText, 0, maxLogSize);
+        int maxSize = maxLogSize == null ? LogSpecifier.UNLIMIT : maxLogSize.intValue();
+        if (maxSize > LogSpecifier.UNLIMIT && entityText != null && entityText.length() > maxSize) {
```

Comment:
`maxSize > LogSpecifier.UNLIMIT` ez olvasva kicsit értelmetlennek tűnik. Mit jelent az, hogy a méret nagyobb, mint határtalan? :) Szerintem jobb lenne itt a `!=` operátor.

Reply:
az UNLIMIT egy -1 értek, és az a alap az annotációban. Ha != engednénk akkor semmi sem gátolja meg hogy negatív értékeket adjon bele, és fura működése lessz. Bele írhattam volna `maxSize > -1` -et is, de így jobban látni hogy mihez van hasonlítva